docs/index.rst

```diff
@@ -178,7 +178,7 @@ out::
     pybabel init -l en_US -d . -i javascript.pot -D javascript
     perl -pi -e 's/fuzzy//' en_US/LC_MESSAGES/javascript.po
     pybabel compile -d . -D javascript
-    open http://0:8000/en-US/jsi18n/
```

Comment:
I'm not sure what to put here. I think having people map a domain eg: amo.dev in /etc/hosts is the way to go. When we implement firefox accounts, this will become required.